rpm_spec/manageiq.spec.in

```diff
@@ -87,6 +87,10 @@ cd %{_builddir}
 %{__cp} -r %{gemset_builddir}/vmdb %{buildroot}%{gemset_root}
 install -m644 %{gemset_builddir}/enable %{buildroot}%{gemset_root}
 
+#copy systemd unit files
+%{__cp} %{appliance_builddir}/systemd_units/* %{buildroot}/usr/lib/systemd/system
```

Comment:
1. We need to come up with a good name, not sure I like `/systemd_units` in the root of the repo, maybe `/build/systemd_units` ?
2. I need to check if the directory exists
3. We're copying _everything_ in root to `/var/www/miq/vmdb` above so these would be included, we probably need to either move these and do this first, or not define the core systemd units in the core repo but instead in manageiq-appliance

Reply:
If we are expecting plugins to eventually possibly bring systemd unit files, then I think core should act consistent to the plugins with respect to the directories.  So, I think we should ignore 3.

Not sure about the file name.  `/build/systemd_units` seems pretty nice

Reply:
:+1: